### DISCUSSION TITLE
Font awesome icons

### DISCUSSION THREAD

**Discussion by boehmerm** [bootstrap-email/bootstrap-email#179] [Ideas]
Hi there,
how would it be possible to you font awesome icons like i.e. icon-ok-sign in the email templates?

Thank you

**Comment by stuyam** [comment]
I have thought about this a bunch, since font awesome is fonts / svgs they don't have good support in email. Rather they would need to be rendered as an image format like png. If you want to do this I would recommend converting the svg to a png using some online converted. However I considered at length creating an icon image CDN that allows you to specific icon, color, and size and have it load the icon as a png. 

How receptive to something like this would people be?

**Reply by MaaxGr** [reply]
Have thought about creating that dynamic image CDN as well. Really looking forward adding icons to my emails in a simple way.
